test/params_conversion_test.c

```diff
@@ -9,13 +9,19 @@
  */
 
 #include <string.h>
-#include <inttypes.h>
 #include <openssl/params.h>
 #include "testutil.h"
 
-#ifdef OPENSSL_SYS_WINDOWS
-# define strcasecmp _stricmp
-#endif
+/* On machines that dont support <inttypes.h> just disable the tests */
+# if !defined(PRId8)
+# define SKIP_TESTS 1
+# endif
+
+#if !defined(SKIP_TESTS)
+
+# ifdef OPENSSL_SYS_WINDOWS
```

Comment:
Is there a reason `#include "e_os.h"` isn't used in the tests?
Anyone?

Reply:
Are you happy with this solution. All the CI loops passed which indicates inttype was available for all of them.

Reply:
Ah, good reminder @paulidale... Except that the correct inclusion is `<openssl/e_os2.h>`. That will include inttypes.h on platforms where we know stdint.h is or may not be available, and stdint.h otherwise.

With that, there should be no need to skip this test.

Reply:
including testutil.h includes e_os2.h.
The problem is the usage of strtoimax in this test (which is why inttypes.h is needed). If inttypes is not available then the tests are skipped.

Reply:
Gotcha